tests/test_rebase_consensus.py

```
@@ -0,0 +1,893 @@
+    @patch("sova.config.loader.load_config", side_effect=Exception("bad config"))
+    def test_returns_defaults_on_failure(self, _mock: MagicMock) -> None:
+        models, threshold, templates, timeout = _load_consensus_config(Path("/fake"))
+        assert models == []
+        assert threshold == 0.66
+        assert templates == {}
+        assert timeout is None
+
+
+class TestCreateProviders:
+    @patch("sova.llm.litellm_provider.LiteLLMProvider")
+    @patch("sova.llm.litellm_provider._HAS_LITELLM", True)
```

Ruff (0.16.1)

[warning] Boolean positional value in function call

(FBT003)